### DISCUSSION TITLE
next auth login funktioniert nicht Production Deployment

### DISCUSSION THREAD

**Discussion by nikstaaar** [neuefische/questions#210] [Web] [answered]
Hallo zusammen, 
wie oben schon beschrieben, funktioniert der login mit nextAuth über gitHub bei mir nicht im Vercel production deployment. Im Preview Modus und auch im Localhost funktioniert es. Ich weiß man muss irgendwie Vercel und nextAuth verbinden, weiß aber nicht wie. 

hier der Link zu Vercel: [capstone-project-a1u6qjbkh-nikstaaar.vercel.app](https://vercel.com/nikstaaar/capstone-project/9XHXusjZi7n39kHpd2x9Xy9LhZRM)

hier mein PR: https://github.com/nikstaaar/capstone-project/pull/14

vielen Dank schonmal im Voraus.

**Comment by mbosselmann** [comment, marked as answer]
Hallo Niklas,
hast du die Schritte zu deinem Deployment durchgeführt, die Ralf in seinem Workshop erklärt hat? Wenn du in dem zweiten Video etwa ab Minute 52 nachschaust, kannst du die Schritte nochmal durchgehen:
- bei GitHub das Vercel Deployment als GitHub App anlegen
- Environment Variablen bei Vercel hinterlegen
- für das Testen von Funktionalitäten für Vercel Preview-Deployments hat Ralf noch als Lösung einen Dummy-Login vorgeschlagen, schaue dir dafür das [Repo von Ralf](https://github.com/actyralf/auth-demo) und die [Antwort von Ralf zur Frage von Omer](https://github.com/orgs/neuefische/discussions/193) an.

Haben dir diese Schritte weitergeholfen? :)

**Reply by nikstaaar** [reply]
Hallo, ich habe die Schritte angewendet, kriege aber immer noch dieselbe Fehlermeldung. :(

**Comment by nikstaaar** [comment]
Das Problem war noch ein `/` zu viel in meinem domain Namen bei Github. Jetzt Funktioniert's. Danke nochmal.
